edxval/api.py

```diff
@@ -104,6 +105,36 @@ def create_video(video_data):
         raise ValCannotCreateError(serializer.errors)
 
 
+def update_video(video_data):
+    """
+    Called on to update Video objects in the database
+
+    update_video is used to update Video objects.
+    Args:
+        edx_video_id (String)
+        data (dict):
```

Comment:
should be `video_data`. Would you mind updating the doc in  `create_video` above it :)